src/Styleguide/Pages/Fixtures/Artworks.ts

```diff
@@ -0,0 +1,53 @@
+import { SingleFollowedArtist } from "./Artists"
```

Comment:
I think it would be cool to identify real artwork that will be very good for testing and have this data reflect the artwork state from production.